install_dependencies.sh

```diff
@@ -63,13 +68,18 @@ if echo $REPLY | grep -E '^[Yy]$' > /dev/null; then
                 fi
 
                 # packages added here should also be added to the Dockerfile
-                sudo apt-get install --no-install-recommends -y autoconf bash-completion bc clang-9 clang-format-9 clang-tidy-9 cmake curl g++-9 gcc-9 gcovr git graphviz libhwloc-dev libncurses5-dev libnuma-dev libnuma1 libpq-dev libreadline-dev libsqlite3-dev libtbb-dev lld-9 man parallel postgresql-server-dev-all python3 python3-pexpect systemtap systemtap-sdt-dev valgrind &
+                sudo apt-get install --no-install-recommends -y autoconf bash-completion bc clang-9 clang-format-9 clang-tidy-9 cmake curl g++-9 gcc-9 gcovr git graphviz libhwloc-dev libncurses5-dev libnuma-dev libnuma1 libpq-dev libreadline-dev libsqlite3-dev libtbb-dev lld-9 man parallel postgresql-server-dev-all python3 systemtap systemtap-sdt-dev valgrind &
 
                 if ! git submodule update --jobs 5 --init --recursive; then
                     echo "Error during installation."
                     exit 1
                 fi
 
+                if ! pip3 install -r requirements.txt; then
+                    echo "Error during installation of python requirements."
+                    exit 1
+                fi
+
```

Comment:
Hm, I don't like that this adds a couple of minutes to the CI's setup step. Pandas and numpy take quite some time to build.

Reply:
How about adding the requirements.txt during the docker build process to the container, installing the requirements in the Dockerfile, and only install them with install_dependencies if in non-`OPOSSUM_HEADLESS_SETUP` mode?

Reply:
If there are components build from source, we should update them. Had the same issues with my compare benchmark script, take a look there.
Precompiled wheels are not available for too old versions.

Reply:
Hmm... The problem with adding the requirements.txt is that we would add a dependency on the Hyrise git to the Docker image. In that case, we could forgo most of the Dockerfile and call install_dependencies right away.

Alternatively, we could only add pexpect to the Dockerfile and skip the requirements.txt using OPOSSUM_HEADLESS_SETUP.

Reply:
Martin's proposal works.

Reply:
Could you explain the current solution please? Doesn't this mean that we install the packages every time?

Reply:
We install the python packages every time but do not build them from source. If we don't want to install them every time, we could do it in the docker file but then 

> The problem with adding the requirements.txt is that we would add a dependency on the Hyrise git to the Docker image

Reply:
Got it. Doesn't seem to add too much to the CI runtime anyway.